CHANGELOG.md

```diff
@@ -7,6 +7,152 @@ documentation before upgrading to a new release.
+  * Don't silently accept authenticated JSON-RPC connections with no valid endpoint. #10415
+  * Sync `Notification#notified_problem_users` across the cluster to prevent lost recovery notifications. #10380
+  * Remove superfluous `)` from a HTTP request log message. #9966
+  * Disable TLS renegotiation (handshake on existing connection) on OpenBSD as well. #9943
+  * Log also the underlying error message when a HTTP request is closed with `No data received` by Icinga 2. #9928
+  * Fix a deadlock triggered by concurrent `/v1/actions/add-comment` and `/v1/actions/acknowledge-problem` requests on
+    the same checkable, as well as a crash that might occur when running perfectly timed `/v1/actions/add-comment`
+    and `/v1/actions/remove-comment` requests targeting the same comment. #9924
+* Icinga DB
+  * Fix missing acknowledgement and flapping history entries due to a number overflow. #10467
+  * Send downtime `cancel_time` only if it is cancelled. #10379
+  * Send only the necessary data to the `icinga:stats` Redis® stream. #10359
+  * Remove a spin lock in `RedisConnection#Connect()` to avoid busy waiting. #10265
+* Writers
+  * Serialize all required metrics before queueing them to a `WorkQueue`. #10420
+  * `OpenTsdbWriter`: Include checkable name in log messages to ease troubleshooting. #10009
+  * `OpenTsdbWriter`: Don't send custom empty tags. #7928
+  * `InfluxDBWriter`: Add missing closing quote in validation error message. #10174
+
+### ITL
+
+* Add `--maintenance_mode_state` (`$vmware_maintenance_mode_state`) argument to `vmware-esx-command` check command. #10435
+* Add `-n` (`$load_procs_to_show$`) argument to `load` check command. #10426
+* Add `--inode-perfdata` (`$disk_np_inode_perfdata$`) argument to `disk` check command. #10395
+* Add `-r` (`$ssh_remote_version$`) and `-P` (`$ssh_remote_protocol$`) arguments to `ssh` check command. #10283
+* Add `--unplugged_nics_state` (`$vmware_unplugged_nics_state$`) argument to `vmware-esx-soap-host-net` and `vmware-esx-soap-host-net-nic` check commands. #10261
+* Add `-X` (`$proc_exclude_process$`) argument to `procs` check command. #10232
+* Add `--dane` (`$ssl_cert_dane$`) argument to `ssl_cert` check command. #10196
+* Fix `check_ssl_cert` deprecation warnings. #9758
+* Fix `check_systemd` executable name add add all missing arguments. #10035
+* Add `-M` (`$snmp_multiplier$` & `$snmpv3_multiplier$`) argument to `snmp` and `snmpv3` check commands. #9975
+* Add `--continue-after-certificate` (`$http_certificate_continue$`) argument to `http` check command. #9974
+* Add `--ignore-maximum-validity` (`$ssl_cert_ignore_maximum_validity$`) argument to `ssl_cert` check command. #10396
+* Add `--maximum-validity` (`$ssl_cert_maximum_validity$`) argument to `ssl_cert` check command. #9881
+* Add `--url` (`$ssl_cert_http_url$`) argument to `ssl_cert` check command. #9759
+* Add `fuse.sshfs` and `fuse.*` (supported only by Monitoring Plugins) to the list of default disk exclude types. #9649
```

Comment:
That must be the wrong PR ID, as that's not merged and something completely different.

* #9649